### ISSUE TITLE
What is seq_len used for?

### ISSUE THREAD

**Issue by wada-s** [mkotha/WaveRNN#8]
Hi !
Thanks for sharing this repo!

I use your script as a vocoder implementation.
I'm reading it but I don't know what seq_len is used for.
Usually, I think that it is used as the number of samples put in one batch.

**Comment by wada-s** [comment]
If I use your script as a vocoder, consider the following to specify the number of frames to be included in one batch. (In this case 16 frames are specified)

`trn_loader = DataLoader(dataset, collate_fn=lambda batch: env.collate(0, 16, 0, batch), batch_size=batch_size, num_workers=2, shuffle=True, pin_memory=True)`

Is this because you use 960 samples for learning like a paper?
